cmd/flux/bootstrap.go

```diff
@@ -58,6 +58,7 @@ var (
 	bootstrapArch               = flags.Arch(defaults.Arch)
 	bootstrapLogLevel           = flags.LogLevel(defaults.LogLevel)
 	bootstrapRequiredComponents = []string{"source-controller", "kustomize-controller"}
+	bootstrapAllComponents      = []string{"source-controller", "kustomize-controller", "helm-controller", "notification-controller", "image-reflector-controller", "image-automation-controller"}
```

Comment:
Please see https://github.com/fluxcd/flux2/blob/main/pkg/manifestgen/install/options.go#L44 we could reuse the array from that package instead of duplicating the components name.

Reply:
It seems the array is incomplete and it seems like updating it will install the extra components by default.

Reply:
You could add the extra components as field in manifestgen then merge the 2 array both here and in the install cmd.